flask/datahandle.py

```diff
@@ -58,9 +62,48 @@ def getCoursesAndItems(data):
                     returndata += getItemsInCourse(data['menu']['meal']['course'], value)
     return returndata
 
+#Formatting list of possible matches into more natural sentence structure
+def findItemFormatting(possiblematches):
+
+    for i in range(len(possiblematches)):
+        if i == 0:
+            continue
+        words = possiblematches[i].split()
+        if(possiblematches[i].split()[-1] == possiblematches[i - 1].split()[-1]):
```

Comment:
Could you add a few code comments what's happening here?

There's some constants " + 8" and some logic that isn't clear to me just by reading this. 

Thanks!

Reply:
Yep, sorry that was just a formatting technicality to make the response look more natural. I'll add a comment explaining.